app/retail/templates/shared/footer_scripts.html

```diff
@@ -16,6 +16,8 @@
 
 {% endcomment %}
 {% load static %}
+<script type="text/javascript" src="https://cdn.jsdelivr.net/gh/ethereum/web3.js@0.20.6/dist/web3.min.js"></script>
```

Comment:
Can you move this into the repo itself? We already have `0.20.7` in the repo itself at `app/assets/v2/js/lib` if that will work for this patch.

Reply:
shared/shared_scripts.html is included in this file at the bottom, why adding it here again?

Reply:
Yep, you're right i forgot remove this line after finish some tests.